pydocx/models.py

```diff
@@ -120,10 +121,25 @@ class ParkingLot(XmlModel):
     def __init__(self, *types, **kwargs):
         default = kwargs.pop('default', [])
         super(XmlCollection, self).__init__(self, default=default)
-        self.types = set(types)
+        self._types = types
         self._name_to_type_map = None
 
     @property
+    def types(self):
+        return set(self._set_types(*self._types))
+
+    def _set_types(self, *types):
+        base_path = 'pydocx.openxml.{0}'
+        for _type in types:
```

Comment:
I really don't like moving all of the imports to be string-based. That removes some of the developer-friendly niceness of immediately knowing when you typo'd an import. Django's approach of mostly using actual imports, but supporting string-based imports where necessary, seems like the best of both worlds. We use strings when we need it, and get immediate feedback on import fails for all of the other cases.